client_int_test.go

```diff
@@ -116,7 +116,7 @@ func TestDeploy(t *testing.T) {
 	}
 	defer del(t, client, "deploy")
 
-	if err := client.Deploy(context.Background(), "."); err != nil {
+	if _, err := client.Deploy(context.Background(), "."); err != nil {
```

Comment:
this `Deploy()` function really returns only one result, not tuple

Reply:
My mistake